l10n_ch_zip/models/city_zip.py

```diff
@@ -14,15 +14,28 @@ class ResBetterZip(models.Model):
     http://www.swisspost.ch/post-startseite/post-adress-services-match/post-direct-marketing-datengrundlage/post-match-zip-factsheet.pdf
     """
 
-    _inherit = 'res.better.zip'
+    _inherit = 'res.city.zip'
 
-    active = fields.Boolean(string='Active', default=True)
-    onrp = fields.Char(string='Swiss Post classification no. (ONRP)',
-                       size=5, help="Primary Key")
-    zip_type = fields.Char(string='Postcode type', size=2)
-    additional_digit = fields.Char(string='Additional poscode digits', size=2)
+    active = fields.Boolean(
+        string='Active',
+        default=True,
+    )
+    onrp = fields.Char(
+        string='Swiss Post classification no. (ONRP)',
+        size=5,
+        help="Primary Key",
+    )
```

Comment:
I don't how this field is used later, do you think it deserves an index?

Reply:
IIRC, this field is there to synchronize the data from swisspost data. So it is not for a daily usage but for once in a while we update the data.